run_tests.sh

```diff
@@ -187,6 +190,18 @@ else
 	fi
 	rm gnl_output.txt
 
+	cp ../get_next_line.c fake_get_next_line.c
+	cp ../get_next_line_utils.c fake_get_next_line_utils.c
```

Comment:
You may need to use the path variable in case user's gnl is in a different directory

```suggestion
	cp ${PATH_GNL}/get_next_line_utils.c fake_get_next_line_utils.c
```

Reply:
Yeah I shouldn't have copy pasted that directly from my test script. Woops.